utils/utils.go

```diff
@@ -10,10 +10,22 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/blang/semver/v4"
 	"github.com/kong/go-kong/kong"
 )
 
-var kongVersionRegex = regexp.MustCompile(`^\d+\.\d+`)
+var (
+	kongVersionRegex = regexp.MustCompile(`^\d+\.\d+`)
+	pathRegexPattern = regexp.MustCompile(`[^a-zA-Z0-9._~/%-]`)
```

Comment:
Please run this regex by Javier.

Reply:
@javierguerragiraldez can you please take a look at this?

Reply:
sure, it's the same as used in Koko and Kong.